includes/transformer/class-factory.php

```diff
@@ -110,6 +110,8 @@ public static function get_transformer( $data ) {
 					return new User( $data );
 				}
 				break;
+			case 'WP_Term':
+				return new Tag( $data );
```

Comment:
```suggestion
				return new Term( $data );
```

Should we call it term to be more universally applicable?